internal/iam/command_acl_delete.go

```diff
@@ -53,7 +53,7 @@ func (c *aclCommand) delete(cmd *cobra.Command, _ []string) error {
 	if len(bindings) > 1 {
 		promptMsg = fmt.Sprintf(pacl.DeleteACLConfirmMsg, resource.Plural(resource.ACL))
 	}
-	if err := deletion.ConfirmDeletionYesNo(cmd, promptMsg); err != nil {
+	if err := deletion.ConfirmPromptYesNo(cmd, promptMsg); err != nil {
```

Comment:
```suggestion
	if err := deletion.PromptYesOrNo(cmd, promptMsg); err != nil {
```

Reply:
`ConfirmPromptYesOrNo` it has a `confirm` step in there

Reply:
The "confirm step" is where the user is prompted, so I think confirm prompt is a bit redundant